docs/guide/setup/chemical_systems_and_thermodynamic_cycles.rst

```diff
@@ -1,6 +1,85 @@
-.. _userguide_chemical_systems:
+.. _userguide_chemicalsystems_and_components:
 
-ChemicalSystems and thermodynamic cycles
-========================================
+ChemicalSystems, Components and Thermodynamic Cycles
+====================================================
 
-issue #759
+Chemical Systems
+----------------
+
+In order to define the input systems to a Protocol,
+which correspond as the end states of an alchemical transformation,
+we need an object model to represent their chemical composition.
+In ``openfe`` a :class:`.ChemicalSystem` is used to capture this information,
+and represents the chemical models that are present in each end state.
+
+A :class:`.ChemicalSystem` **does** include information, where present, on:
+
+* exact atomic information (including protonation state) of protein, ligands, co-factors, and any crystallographic
+  waters
+* atomic positions of all explicitly components such as ligands or proteins
```

Comment:
What is an "explicitly component"? Maybe explicitly defined components?

Reply:
yep fixed